vuepress/docs/next/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -522,31 +453,33 @@ above would look like this:
           // note that property's fields are not mentioned - defaulting them to 12 columns in are that is available to them
         },
       },
-    };
+    }
 
-    As Material UI components are used for field templates, there might be a need to pass some Material UI options into the field. You can do that by adding muiProps object to the ui:options.
-    For example, if you would like to make the Down payment field resizable, you can add multiline: true to muiProps option. If you want to make the field take up multiple rows by default, add rows and rowsMax fields. The latter limits how many rows should be added until the scroll bar is shown.
-    downpayment: {
-      'ui:options': {
-        size: 4,
-        muiProps: {
-          multiline: true,
-          rows: 2,
-          rowsMax: 4
+As Material UI components are used for field templates, there might be a need to pass some Material UI options into the field. This can be done by adding the `muiProps` object to `ui:options`.
+
+For example, if you would like to make the down payment field resizable, you can add `multiline: true` to the `muiProps` option. If you want to make the field to take up multiple rows by default, add the fields `rows` and `rowsMax`. The latter limits how many rows can be added until the scroll bar is shown.
```

Comment:
'the field take up multiple'

Reply:
yikes! pretty sure that's what i meant and never finished editing :| good catch